indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueryTool.java

```diff
@@ -85,28 +86,12 @@ public TaskQueryTool(
   }
 
   /**
-   * @param lockFilterPolicies Requests for conflicing lock intervals for various datasources
-   * @return Map from datasource to intervals locked by tasks that have a conflicting lock type that cannot be revoked
+   * @param lockFilterPolicies Requests for conflicing locks for various datasources
+   * @return Map from datasource to conflicting lock infos
    */
-  public Map<String, List<Interval>> getLockedIntervals(List<LockFilterPolicy> lockFilterPolicies)
+  public Map<String, List<TaskLockInfo>> getConflictingLockInfos(List<LockFilterPolicy> lockFilterPolicies)
```

Comment:
Rename all the methods named `getConflictingLockInfos()` to `getActiveTaskLocks()`.
These APIs are just returning active locks which satisfy certain conditions.

The fact whether they conflict with something else or not is the caller's concern.